### PR TITLE
Audio Poster issue fix

### PR DESCRIPTION
**Pull request by ghost**
React-native-video support audio only play. In case of audio play we can supply the poster and React-native-video will show the poster. But after the audio has started playing the poster is hidden and there is blank view. This PR is to show the audio poster even after the audio starts playing

### CODE REVIEW

File: react-native-video.podspec [hunk 1]
```diff
@@ -9,8 +9,8 @@ Pod::Spec.new do |s|
   s.description    = package['description']
   s.license        = package['license']
   s.author         = package['author']
-  s.homepage       = 'https://github.com/brentvatne/react-native-video'
-  s.source       = { :git => "https://github.com/brentvatne/react-native-video.git", :tag => "#{s.version}" }
+  s.homepage       = 'https://github.com/learnyst/react-native-video'
+  s.source       = { :git => "https://github.com/learnyst/react-native-video.git", :tag => "#{s.version}" }
```
Comment (CHaNGeTe) [inline react-native-video.podspec]: please change this for the PR

**Review by CHaNGeTe — CHANGES_REQUESTED** [review]

File: react-native-video.podspec (hunk 1 above)
Comment (jensellfors) [reply react-native-video.podspec]: Should we not just change to https://github.com/react-native-community/react-native-video? Don't know why brentvatne is still there

**Review by jensellfors — CHANGES_REQUESTED** [review]

File: react-native-video.podspec (hunk 1 above)
Comment (sridhard) [reply react-native-video.podspec]: Yes we need to change...I will change it

**Comment by sridhard** [comment]
@CHaNGeTe @cobarx @jenshandersson can you please review and merge the PR

**Review by jensellfors — APPROVED** [review]

File: react-native-video.podspec (hunk 1 above)
Comment (bubah) [reply react-native-video.podspec]: Any idea when this fix will be complete?

**Review by benoitdion — APPROVED** [review]

File: react-native-video.podspec (hunk 1 above)
Comment (bubah) [reply react-native-video.podspec]: Hi, Any idea when a new release  that includes this fix will be made available?

Comment (bubah) [reply react-native-video.podspec]: Hello, My group at work is using react-native-video in one of our apps and this issue is currently one of the blockers keeping us from releasing. Would you be able to provide an ETA on when the next version that includes this fix will be released?

**Pull request merged**